rotkehlchen/externalapis/beaconchain/service.py

```diff
@@ -238,7 +244,7 @@ def get_validator_data(
         """Returns data for the given validators
 
         Essentially calls:
-        https://beaconcha.in/api/v1/docs/index.html#/Validator/get_api_v1_validator__indexOrPubkey_
+        https://beaconcha.in/api/v1/docs/index.html#/Validator/post_api_v1_validator
```

Comment:
please add a small test to confirm that exited validators aren't queried anymore and also as a regression test